rstudio-tidyverse/rockcraft.yaml

```
@@ -0,0 +1,259 @@
+
+name: rstudio-tidyverse
+summary: An image for using tidyverse
+description: |
+  The tidyverse is an opinionated collection of R packages designed for data science. 
+  Tidyverse are installed in Conda environment, which is automatically activated.
+version: "v1.10.0-rc.1"
+license: Apache-2.0
+base: ubuntu@22.04
+run-user: _daemon_
+
+platforms:
+  amd64:
+
+services:
+  rstudio-tidyverse:
+    override: replace
+    summary: "Rstudio Tidyverse service"
+    startup: enabled
+    command: /usr/lib/rstudio-server/bin/rserver --server-daemonize=0 --server-data-dir=/tmp/rstudio_runtime --server-working-dir=/home/_daemon_ --server-user=_daemon_ --rsession-which-r=/opt/conda/lib/R --www-address=0.0.0.0 --www-port=8888 --auth-none=1 --rsession-path=/opt/rsession.sh --rsession-ld-library-path=/opt/conda/lib
+    working-dir: /home/
+    environment:
+       NB_USER: _daemon_
+       NB_UID: 1000
+       NB_GID: 0
+       NB_PREFIX: /
+       USERS_GID: 100
+       HOME: /home/_daemon_
+       HOME_TMP: /tmp_home/_daemon_
+       USER: _daemon_
+       SHELL: /bin/bash
+       LANG: en_US.UTF-8to
+       LANGUAGE: en_US.UTF-8
+       LC_ALL: en_US.UTF-8
+       PATH: /opt/conda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+       PYTHONPATH: /opt/conda/lib/python3.11/site-packages/
+       RSTUDIO_RUNTIME_DIR: "/tmp/rstudio_runtime"
+       XDG_RUNTIME_DIR: "/tmp/runtime-_daemon_"
+parts:
+  security-team-requirement:
+    plugin: nil
+    override-build: |
+      mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
+      (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && \
+      dpkg-query -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) \
+      > ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
+
+  overlay-pkgs:
+    plugin: nil
+    overlay-packages:
+      - apt-transport-https
+      - bash
+      - bzip2
+      - ca-certificates
+      - curl
+      - git
+      - gnupg
+      - gnupg2
+      - locales
+      - lsb-release
+      - nano
+      - software-properties-common
+      - tzdata
+      - unzip
+      - vim
+      - wget
+      - xz-utils
+      - zip
+
+  kubectl:
+    plugin: nil
+    stage-snaps:
+      - kubectl/1.31/stable
+    organize:
+      kubectl: bin/kubectl
+    stage:
+      - bin/kubectl
+
+  generate-locale:
+    plugin: nil
+    build-packages:
+      - locales
+    override-build: |
+      echo "en_US.UTF-8 UTF-8" > /etc/locale.gen
+      locale-gen
+      mkdir -p $CRAFT_PART_INSTALL/usr/lib/locale $CRAFT_PART_INSTALL/usr/share/i18n/locales
+      cp /usr/lib/locale/locale-archive $CRAFT_PART_INSTALL/usr/lib/locale/locale-archive
+      cp -r /usr/share/i18n/locales $CRAFT_PART_INSTALL/usr/share/i18n/locales
+
+  rstudio-tidyverse:
+    plugin: nil
+    source: https://github.com/kubeflow/kubeflow
+    source-type: git
+    source-tag: v1.10.0-rc.1
```

suggestion: Let's use the latest tag
```suggestion
    source-tag: v1.10.0
```

---

Fixed